projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,6 +18,53 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const ConnectionItem = props => {
+	return props.connectionIds.map( connectionId => {
+		const connection = jetpackSocial.connections[ props.provider ][ connectionId ];
+		return (
+			<div>
```

Comment:
We might want to set [a `key`](https://reactjs.org/docs/lists-and-keys.html#keys) on each of these, but we could deal with things like that when we adjust the markup for the final design.